<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ip_BulkSend</class>
 <widget class="QWidget" name="ip_BulkSend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>481</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>IP三相</string>
   </property>
   <widget class="QSpinBox" name="tpeNum">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>48</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>50</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>模拟设备数量:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>151</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>起始IP地址：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="devIp">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>151</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>172.18.1.2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bulkSendBtn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>248</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始发送</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>56</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>电流规格：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>250</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>时间间隔:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Ttimeinv">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>248</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="suffix">
     <string> 秒</string>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>257</x>
      <y>157</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>主副机数量:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Taddnum">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="TcurCap">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>50</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="suffix">
     <string> A</string>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
    <property name="value">
     <number>63</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>IP单相</string>
   </property>
   <widget class="QSpinBox" name="speNum">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>49</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>49</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>模拟设备数量:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Saddnum">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>130</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>131</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>起始IP地址：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Stimeinv">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>209</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="suffix">
     <string> 秒</string>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="SdevIp">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>131</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>172.17.1.2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>136</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>主副机数量:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>216</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>时间间隔:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>49</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>电流规格：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SbulkSendBtn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>213</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始发送</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="ScurCap">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>50</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="suffix">
     <string> A</string>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
    <property name="value">
     <number>63</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>speNum</tabstop>
  <tabstop>SdevIp</tabstop>
  <tabstop>Stimeinv</tabstop>
  <tabstop>ScurCap</tabstop>
  <tabstop>Saddnum</tabstop>
  <tabstop>SbulkSendBtn</tabstop>
  <tabstop>tpeNum</tabstop>
  <tabstop>devIp</tabstop>
  <tabstop>Ttimeinv</tabstop>
  <tabstop>TcurCap</tabstop>
  <tabstop>Taddnum</tabstop>
  <tabstop>bulkSendBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
